### ISSUE TITLE
Content is not being save while using `grammarly` as extension.

### ISSUE THREAD

**Issue by wotta** [codex-team/editor.js#755]
Today I noticed some strange behavior when using editorjs with the following plugins.

```json
"@editorjs/editorjs": {
    "version": "2.13.0",
    "resolved": "https://registry.npmjs.org/@editorjs/editorjs/-/editorjs-2.13.0.tgz",
    "integrity": "sha512-oeYsIRvm0bSUJsggZK0marTYfatXILhESoqshwGnPCinaEbA1VAAWfc74saMLuTmujyefgQE2qYjHpKyZNM0mQ==",
    "dev": true,
    "requires": {
        "codex-notifier": "^1.1.2"
    }
},
"@editorjs/embed": {
    "version": "2.2.0",
    "resolved": "https://registry.npmjs.org/@editorjs/embed/-/embed-2.2.0.tgz",
    "integrity": "sha512-jWjFL6VyiGlezSWQRvddMHpidGHy2blUS9lmtNxuIZRwob6Xizg8LweKSFEvjr9tPLqKyX3sIeSQlnCb1vntJQ==",
    "dev": true
},
"@editorjs/header": {
    "version": "2.2.3",
    "resolved": "https://registry.npmjs.org/@editorjs/header/-/header-2.2.3.tgz",
    "integrity": "sha512-eFPgsT/OJA7HxCl4idrH1ccZEkmVZWfEmwo7DysOAb2DI+5/IcBIVZbPv6EapjL6MOEHKzON8lCI4jQOaBJlLw==",
    "dev": true
},
"@editorjs/image": {
    "version": "2.3.0",
    "resolved": "https://registry.npmjs.org/@editorjs/image/-/image-2.3.0.tgz",
    "integrity": "sha512-KoGPtpdpS5Wbjz/peHa8oy53C89gDATADSNae3hfl89DzttToWKNiSf7T4IVmYvqa4k1+FRpe0KRSlCzWxYIlA==",
    "dev": true
},
"@editorjs/link": {
    "version": "2.1.2",
    "resolved": "https://registry.npmjs.org/@editorjs/link/-/link-2.1.2.tgz",
    "integrity": "sha512-hIMMYFNU2AIdF0KRrlSGzsMDeiSoo4/AXxA9FkIRerMyNIoBRENK56UpCdWDnDveW0hkNvWlO7dKG/VwPyifnw==",
    "dev": true,
    "requires": {
        "@babel/runtime": "^7.3.1"
    }
},
"@editorjs/list": {
    "version": "1.3.4",
    "resolved": "https://registry.npmjs.org/@editorjs/list/-/list-1.3.4.tgz",
    "integrity": "sha512-hj/yJXUjpAsh946js4zK0eHTnKJOiXXg6wpvTisQbQ3sjtKmUdS/KAyvihKhkKMa1y8SDR3L4K/3raPryyLC1A==",
    "dev": true
},
"@editorjs/paragraph": {
    "version": "2.5.0",
    "resolved": "https://registry.npmjs.org/@editorjs/paragraph/-/paragraph-2.5.0.tgz",
    "integrity": "sha512-73HwzALLcEhgyxd7j0g2v8a21GfEDBhk/aCifYMRoygf6gk3/1wBJO0DmnUZVcKwdRgXmHOWRjY5yyCCe1NnDQ=="
},
"@editorjs/table": {
    "version": "1.2.0",
    "resolved": "https://registry.npmjs.org/@editorjs/table/-/table-1.2.0.tgz",
    "integrity": "sha512-pCcNf/9JzVWQw6TsqUHew73UoVSYTftXviKVqsnVazx4/D8eH5g5fsEY5L0LEl9KC0mJMJij+h6I1ge70kDlgg==",
    "dev": true
},

```

If I have typed a paragraph and try to save it, it doesn't have the paragraph value.
Instead, it has the value that Grammarly shows in their error counter.

See the following screenshot for more information about what I mean.
![](https://puu.sh/DmPiz/041f134aa9.png)

Is there a way to confine the data that Editor.js takes?

**Comment by nicholasgriffintn** [comment]
You can add data-gramm_editor="false" to inputs to disable Grammarly, not sure if this will work with other elements though.

**Comment by wotta** [comment]
@nicholasgriffintn I tried this and it doesn't seem to work on the main div that Editor.js uses.

I will check if I can add it to the paragraph and heading fields as a custom attribute.

**Comment by neSpecc** [comment]
Resolved in 2.16 #966
